src/main/java/org/breedinginsight/brapps/importer/services/processors/ExperimentProcessor.java

```diff
@@ -293,14 +317,32 @@ private void getNewBrapiData(List<BrAPIImport> importRows, Program program, User
             PendingImportObject<BrAPIObservationUnit> obsUnitPIO = createObsUnitPIO(program, commit, envSeqValue, importRow);
             String key = createObservationUnitKey(importRow);
             this.observationUnitByNameNoScope.put(key, obsUnitPIO);
+
+            for (Column<?> column : phenotypeCols) {
+                PendingImportObject<BrAPIObservation> obsPIO = createObservationPIO(importRow, column.name(), column.getString(i));
+                this.observationByHash.put(getImportObservationHash(importRow, getVariableNameFromColumn(column)), obsPIO);
+            }
         }
     }
 
-    private String createObservationUnitKey(ExperimentObservation importRow) {
+    private static String createObservationUnitKey(ExperimentObservation importRow) {
```

Comment:
Why was this changed to be static?

Reply:
Think I just changed it when I was in there in line with the rationale in the following comment.

Reply:
Pushed changes.